Plugins/NativeInput/Source/Shared/NativeInput.cpp

```diff
@@ -176,20 +200,6 @@ namespace Babylon::Plugins
                 return;
             }
 
-            // If all "buttons" are up, then remove the device (e.g. device "disconnected").
-            for (size_t index = 0; index < deviceInputs.size(); index++)
-            {
-                if (index != POINTER_X_INPUT_INDEX &&
-                    index != POINTER_Y_INPUT_INDEX &&
-                    index != POINTER_DELTA_HORIZONTAL_INDEX &&
-                    index != POINTER_DELTA_VERTICAL_INDEX &&
-                    index != POINTER_MOVE_INDEX &&
-                    deviceInputs[index] > 0)
-                {
-                    return;
-                }
-            }
-
```

Comment:
Why is this not necessary anymore?

Reply:
For Mouse, we return before hitting that and for Touch, we only have one button to zero and it's already zeroed so this does nothing.